docs/reference/embed.rst

```diff
@@ -15,6 +15,8 @@ Single graph embedding
 
 .. autoclass:: AdjacencySpectralEmbed
 .. autoclass:: LaplacianSpectralEmbed
+.. py:class:: CovariateAssistedSpectralEmbed
```

Comment:
use `.. autoclass::` instead of `.. py:class::`

Reply:
Here I'm trying to find a way to just show part of the doc string instead all doc strings in the class.
I only want to show docs for 'class CASC' and 'method fit'. For the others they are something private  and not necessary for people who use this class. And for the ones inhereited from Base Embed like set parameters/get parameters. They are not overrided in casc so I guess they won't work at all.

Reply:
Then make the private functions start with `_`

Reply:
Got it. Will make this change. What about those inherited from BaseEmbed but never expected to be used?